src/tests/request_values_tests.rs

```diff
@@ -34,6 +34,21 @@ pub fn tests_from_value_get_type(test: String) -> ComplexType {
         "TinyInt" => (CqlType::TinyInt, None, None),
         "Time" => (CqlType::Time, None, None),
         "Timeuuid" => (CqlType::Timeuuid, None, None),
+        "Tuple" => {
+            return ComplexType::two_support(
+                CqlType::Tuple,
+                Some(ComplexType::simple_type(CqlType::Text)),
+                Some(ComplexType::two_support(
+                    CqlType::Tuple,
+                    Some(ComplexType::simple_type(CqlType::Int)),
+                    Some(ComplexType::two_support(
+                        CqlType::Tuple,
+                        Some(ComplexType::simple_type(CqlType::Empty)),
+                        None,
+                    )),
+                )),
+            )
+        }
```

Comment:
I have absolutely never seen a tuple represented as a tree. @PiotrJunior, you have unique talent of bringing up very extraordinary solutions.